ansible-scylla-node/tasks/common.yml

```diff
@@ -299,6 +299,63 @@
       when: token_distributor is defined
   when: start_scylla_service is defined and start_scylla_service|bool
 
+- name: Check if authentication is enabled
+  lineinfile:
+    path: /etc/scylla/scylla.yaml
+    regexp: '^(authenticator: (.*PasswordAuthenticator.*|.*TransitionalAuthenticator.*))|(authorizer: (.*CassandraAuthorizer.*|.*TransitionalAuthorizer.*))'
+    state: absent
+  check_mode: yes
+  register: _authentication_enabled
+  run_once: true
+
+- name: If authentication is enabled, adjust replication for system_auth keyspace
+  block:
+    - fail:
+        msg: "Unable to adjust system_auth replication unless start_scylla_service is set to True"
+      when: start_scylla_service == False
+
+    - name: Get datacenter name
+      uri:
+        url: "http://{{ scylla_api_address }}:{{ scylla_api_port }}/snitch/datacenter"
+        method: GET
+      register: _datacenter_out
+      until: _datacenter_out.status == 200
+      retries: 5
+      delay: 1
+
+    - name: Prepare per DC replication_factor list
+      set_fact:
+        dcs_to_rf: "{{ dcs_to_rf | default([]) + [\"'\" + hostvars[item]['_datacenter_out'].json + \"':\" + replication_factor|string] }}"
+      loop: "{{ groups['scylla'] }}"
+      run_once: true
+
+    - name: Adjust replication for system_auth keyspace
+      shell: |
+        cqlsh {{ broadcast_address }} -u {{ cql_username }} -p {{ cql_password }} -e "ALTER KEYSPACE system_auth WITH replication = {'class': '{{ replication_strategy }}', {{ dcs_to_rf | unique | join(',') }}};"
+      run_once: true
+
+    - name: Cleanup system_auth
+      async_task:
+        shell: |
+          nodetool cleanup system_auth
+        alias: scylla_cleanup_system_auth
+        async: "{{ cleanup_timeout_seconds }}"
+        retries: "{{ cleanup_timeout_seconds // 30 }}" # retries = cleanup_timeout_seconds / delay
+        delay: 30
+      register: _cleanup_output
+
+    - name: Cleanup logs
+      debug: var=_cleanup_output
```

Comment:
What is this about?

Reply:
Just watching the cleanup logs, in the same way we do for the `run_decomission` and `run_cleanup` playbooks.